src/SerializerBuilder.php

```diff
@@ -96,18 +102,25 @@ class SerializerBuilder
      */
     private $accessorStrategy;
 
-    public static function create()
+    public static function create(...$args)
     {
-        return new static();
+        return new static(...$args);
     }
 
-    public function __construct()
+    public function __construct(HandlerRegistryInterface $handlerRegistry = null, EventDispatcherInterface $eventDispatcher = null)
```

Comment:
Missing nullable `?`.

Reply:
isnt optional when is an object?

Reply:
I mean, with the current definition, 
```php
new SerializerBuilder();
new SerializerBuilder(null, null);
new SerializerBuilder(null, new EventDispatcher());
```

all of this should be valid, right?

Reply:
Yes, but the signature is still missing the `?`. 😆